### ISSUE TITLE
Cannot Reproducing Pi0_aloha_sim Model by fine-tuning

### ISSUE THREAD

**Issue by zhengwei-www** [Physical-Intelligence/openpi#590]
Hi, I'm trying Reproducing Pi0_aloha_sim Model, the procedure is:

- choose the base model, my choice is Pi0_base
- choose the training dataset, my choice is the official "lerobot/aloha_sim_transfer_cube_human"
- choose the traning norm_stats, I have 3 choices: **1. pi0_base norm_stats, 2. Pi0_aloha_sim norm_stats, 3. norm_stats computed by dataset (lerobot/aloha_sim_transfer_cube_human)**
- start training

Regardless of which norm_stats is used, the fine-tuned model's performance is poor.

I thought the norm_stats from Pi0_aloha_sim and from dataset (lerobot/aloha_sim_transfer_cube_human) should be the same, but the actual result is different. And different norm_stats files influence a lot.

This is the loss curve during training:

<img width="3000" height="3600" alt="Image" src="https://github.com/user-attachments/assets/9b4f289d-9a22-4b55-9d82-8619495d3ce7" />

This is one of the testing result video:

https://github.com/user-attachments/assets/c92fb94b-6f02-4a37-a78c-9f3d7ad9143c

Is there any advice to fine-tune the model? Thanks.
